Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
Because as far as I recall, @nightpool didn't do that on purpose since it gives a popup about Reblog Display Options making them aware of a new extension which has the functionality of Color Quotes and a lot more. Old Stats is a hacky-breaky version of Old Sidebar and should thus not be able to be re-installed.